sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/sweep/sweep_job.py

```diff
@@ -323,7 +327,7 @@ def _load_from_rest(cls, obj: JobBase) -> "SweepJob":
             search_space=_search_space,  # type: ignore[arg-type]
             limits=SweepJobLimits._from_rest_object(properties.limits),
             early_termination=early_termination,
-            objective=properties.objective,
+            objective=Objective._from_rest_object(properties.objective) if properties.objective else None,
```

Comment:
Just curious, how do the previous code works, or it used to fail 100%?

Reply:
it would have never worked, fetching job and updating it.